a.el

```diff
@@ -124,10 +126,12 @@
     (length coll))
 
    ((hash-table-p coll)
-    (length (a-keys coll)))))
+    (hash-table-count coll))))
 
+;; 1) is it ok that collections of different types can be equal?
```

Comment:
Yes, that's by design, the idea being that their "value" is the same (the way a Clojure seq and vector can be equal).

Thinking about it though we could have a separate `a-eql` which checks type, and `a-equal` which is more lenient.